test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MicrosoftDataSqlClientTests.cs

```diff
@@ -42,14 +42,15 @@ public void SubmitsTracesWithNetStandard(string packageVersion, bool enableCallT
             // Once this is fully supported, this will add another 2 complete groups for all frameworks instead
             // of 4 extra spans on net461 and netcoreapp2.0+
 #if NET461
-            var expectedSpanCount = 77; // 7 queries * 11 groups
+            var expectedSpanCount = 133; // 7 queries * 11 groups
```

Comment:
If this is the new number, we should prob update the comment

Reply:
I put off updating these 😅 Done in f9829db87629aa5c357cc4ad5a69a51fcd1da1ad